addons/portal_rating/static/src/xml/portal_tools.xml

```diff
@@ -6,7 +6,6 @@
         <t t-set="empty_star" t-value="5 - (val_integer+Math.ceil(val_decimal))"/>
         <div class="o_website_rating_static"
              t-att-style="inline_mode ? 'display:inline' : ''"
-             t-attf-aria-label="#{Math.round(val * 100) / 100} stars on 5"
```

Comment:
Could your commit message explain all the cases you fix please, as it's not a domain known by a lot of people, it would make sense to describe what was done and avoid saying "etc..." in the commit message :wink: 

For instance here: why was it wrong?